Gemfile

```diff
@@ -33,10 +36,13 @@ gem 'jbuilder', '~> 2.5'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+gem 'haml', '5.0.0.beta.2'
```

Comment:
I don't recall adding that one ... I think it may have been in there when we started the project. Possibly added by Karin?